finta/finta.py

```diff
@@ -2199,5 +2199,42 @@ def EVSTC(
 
         return pd.Series(STOD_DoubleSmooth, name="{0} period EVSTC".format(k_period))
 
+
+    @classmethod
+    def WILLIAMS_FRACTAL(cls, ohlc: DataFrame, period: int = 2) -> DataFrame:
+        """
+        Williams Fractal Indicator
+        Source: https://www.investopedia.com/terms/f/fractal.asp
+
+        :param DataFrame ohlc: data
+        :param int period: how many lower highs/higher lows the extremum value should be preceded and followed.
+        :return DataFrame: fractals identified by boolean
+        """
+
+        def is_bullish_fractal(x):
+            if x[period] == min(x):
+                return 1
```

Comment:
I believe this should be returning True/False.

Reply:
Changed that and added the test